src/js/tech/html5.js

```diff
@@ -86,6 +87,12 @@ class Html5 extends Tech {
    * @method dispose
    */
   dispose() {
+    let tt = this.el().textTracks;
+
+    tt.removeEventListener('change', Fn.bind(this, this.handleTextTrackChange));
```

Comment:
I like the idea of something simple and less async, but whatever you think
works best.

-- 
Sent from mobile